.github/workflows/unit-tests.yml

```diff
@@ -113,13 +113,24 @@ jobs:
         with:
           path: ${{ github.run_id }}
 
+      - name: Check for notebook changes
+        id: check_notebooks
+        run: |
+          if git diff --quiet main HEAD -- '*.ipynb'; then
```

Comment:
Can you comment what this if line is doing? Based on the variable getting set on the next line it's the opposite of what I would expect (looking for a match).

Reply:
Yeah I'm not sure it's working yet 🤣